### PR TITLE
User activity commerce events

### PR DESCRIPTION
**Pull request by paultopher**

### CODE REVIEW

File: mParticle-Button/MPKitButton.m [hunk 1]
```diff
@@ -197,4 +228,22 @@ - (void)observeAttributionTokenDidChangeNotification:(NSNotification *)note {
...
 }
 
+
+- (NSArray<ButtonProduct *> *)buttonProductsFromProducts:(NSArray <MPProduct *> *)products {
+    NSMutableArray *buttonProducts = [NSMutableArray array];
+    for (MPProduct *product in products) {
+        ButtonProduct *buttonProduct = [ButtonProduct new];
+        buttonProduct.name = product.name;
+        buttonProduct.id = product.sku;
+        buttonProduct.value = (NSInteger)(product.price.doubleValue * 100);
```
Comment (najmsheikh) [inline mParticle-Button/MPKitButton.m:238]: Will this throw an exception if `product.price` is unavailable?

**Review by najmsheikh — APPROVED** [review]
Changes LGTM but have you tested this in a test app?

File: mParticle-Button/MPKitButton.m (hunk 1 above)
Comment (paultopher) [reply mParticle-Button/MPKitButton.m:238]: Nope, will end up `0`

**Comment by paultopher** [comment]
> Changes LGTM but have you tested this in a test app?

Tested in FruitParticles, all events reporting properly w/ correct data!

**Pull request merged**